region/tourian/main.json

```diff
@@ -193,7 +223,14 @@
           "to": [
             {
               "id": 2,
-              "requires": null,
+              "requires": [
+                {"or": [
+                  "SpaceJump",
+                  {"cost": [
+                    {"acidFrames": 30}
```

Comment:
In both directions, there's a way to avoid those frames with just Speed Booster as well,